DISDRODB/Raw/EPFL/EPFL_ROOF_2012/metadata/43.yml

```diff
@@ -22,11 +22,11 @@ continent: ''
 latitude: -9999
 longitude: -9999
 altitude: -9999
-deployment_status: ''
+deployment_status: 'terminated'
```

Comment:
Can you fix lat/lon also of this station?

Reply:
Unfortunately i do not know anything about this installation